drivers/console/ipm_console_receiver.c

```diff
@@ -9,7 +9,7 @@
 #include <errno.h>
 
 #include <kernel.h>
-#include <misc/ring_buffer.h>
```

Comment:
This is technically API breakage.
We alright with that?

Reply:
addressed